impress-org/givewp
# Fix: Form Grid strings are translatable and account for singular or plural usages

Resolves #6587

## Description

This fixes and adds translations on several strings. The WP function _n() which translates a string to either a singular or plural version based on a number value was not working correctly. The original container element that was used in the _n() ,  has been moved outside the conditional statement and translations have been simplified by being inserted directly into the html. A Translation has also been added to a "Donation" string that was previously not translated.

## Affects
FormGrid

## Visuals


https://user-images.githubusercontent.com/75056371/196825716-6d36a84b-1748-4a79-9208-625fce1981db.mov

## Testing Instructions

- Create a form and set the donation goal type to either donors or donations.
- Set the goal amount to 1.
- Make a donation.
- Create a page with the FormGrid shortcode or block.
- View the the bottom left & verify "1 of 1 donor or donation"
- Edit the form goal to 25.
- View the bottom left of the Form Grid & verify "1 of 25 donors or donations"

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203211562108141
